src/openlcb/TractionCvSpace.cxx

```diff
@@ -282,7 +282,7 @@ StateFlowBase::Action TractionCvSpace::fill_read1_packet()
      * between long and short addresses */
     if (dccAddress_ >= 0x80)
```

Comment:
Since we have a known bit indicating long/short address, can we move this check up to `TractionCvSpace::set_node(Node *node)` where we can test the node ID?

Reply:
Yes. I will do this in a separate PR though.